README.md

```diff
@@ -59,6 +60,8 @@ If using a collection to produce `<select>` `<option>`s, the following may also
 - `[textAttribute]`: model attribute to use as the text of the option node in the select box
 - `[yieldModel]`: [default: `true`] if options is a collection, yields the full model rather than just its `idAttribute` to `.value`
 
+When the collection changes, the view will try and maintain its currently `.value`.  If the corresponding model is removed, the <select> control will default to the 0th index <option> and update its value accordingly.
```

Comment:
On this page itself it's even borked. https://github.com/cdaringe/ampersand-select-view/tree/4.0.0.  Fixed in master now though.